storm-server/src/main/java/org/apache/storm/LocalCluster.java

```diff
@@ -323,6 +342,13 @@ private static boolean areAllWorkersWaiting() {
         }
     }
 
+    private static final long DEFAULT_ZK_PORT = 2181;
```

Comment:
Nit: Please move this to the other constants so it's not in the middle of method declarations.